scripts/sentinel.sh

```diff
@@ -44,6 +58,7 @@ set_output_names () {
   hms=${date:8:7}
 
   hlsversion="v2.0"
+  fmaskversion="4.7"
```

Comment:
+1 this would be a great feature for the future to save the debug output with a version specific filename

I'm skeptical there's an easy way to get this from the MATLAB code, but we can try to add this in the Fmask 5 Python code (made a comment in the ticket, https://github.com/NASA-IMPACT/hls_development/issues/360)